integration/copy_test.go

```diff
@@ -297,7 +297,7 @@ func (s *CopySuite) TestCopyCompression(c *check.C) {
 		c.Assert(err, check.IsNil)
 
 		assertSkopeoSucceeds(c, "", "--tls-verify=false", "copy", "dir:fixtures/"+t.fixture, t.remote)
-		assertSkopeoSucceeds(c, "", "--tls-verify=false", "copy", t.remote, "dir:"+dir)
+		assertSkopeoSucceeds(c, "", "--tls-verify=false", "copy", t.remote, "dir:"+dir+"::compress")
```

Comment:
See the discussion in https://github.com/containers/image/issues/192 and https://github.com/containers/image/pull/193 — this significantly weakens the test, now we don't really know whether the compression has happened on push or pull.